### PR TITLE
News slider

### PR DESCRIPTION
**Pull request by gitjul**

### CODE REVIEW

File: admin/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 backend:
   name: git-gateway
   # name: test-repo
-  branch: master
+  branch: news-slider
```
Comment (3v0k4) [inline admin/config.yml]: Is this supposed to be changed?

File: assets/js/slider.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 document.addEventListener("DOMContentLoaded", () => {
-  tns({
-    container: ".js-slider",
-    autoHeight: true,
-    speed: 400
+  [].forEach.call(document.querySelectorAll(".js-slider"), function(el) {
```
Comment (3v0k4) [inline assets/js/slider.js]: What do you say about changing `el` to `element`?

File: _includes/slider-init.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<script
+  defer
+  type="text/javascript"
+  src="https://cdnjs.cloudflare.com/ajax/libs/tiny-slider/2.9.2/min/tiny-slider.js"
+></script>
+<script defer type="text/javascript" src="/assets/js/slider.js"></script>
```
Comment (3v0k4) [inline _includes/slider-init.html]: Are we using the same JS libraries / code for all the sliders on HabitatMap?

File: admin/config.yml (hunk 1 above)
Comment (gitjul) [reply admin/config.yml]: yes

File: assets/js/slider.js (hunk 2 above)
Comment (gitjul) [reply assets/js/slider.js]: fine

File: _includes/slider-init.html (hunk 3 above)
Comment (gitjul) [reply _includes/slider-init.html]: yes, same library

**Pull request merged**